mongo/filterstore.go

```diff
@@ -40,6 +41,9 @@ func (s *FilterStore) AddFilter(host string, filter *models.Filter) (*models.Fil
 	session := s.Session.Copy()
 	defer session.Close()
 
+	// Initialise with a timestamp
+	filter.UniqueTimestamp = 1
```

Comment:
You can create a valid mongo timestamp with gedges mgo bson package by doing the following:

`filter.UniqueTimestamp = bson.NewMongoTimestamp(currentTime, 1)`